index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+  <header class="header">
+    <div class="toolbar logo">John Doe</div>
+  <div class="toolbar"><img src="image/Union.png" class="imagcss" alt="dropdown menu"></div></header>
+  <div class="headline">
+<h1 class="primary">Hey There.
+  I’m Ghilain</h1>
+  <h2 class="support">I’am a Software Developer</h2>
+  <p class="description">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+    don’t hestiate to contact me.</p>
+    <img src="image/Social-Media.png" class="social" alt="social media image">
+    <img src="image/illustration-header-mobile.png" class="illustrate" alt="vector image">
```

Comment:
Kindly put the social links inside an  `ul` tag each one with an `li` element, so this will make your code more professional.
(currently you used an image for all of them) 👍😃

Reply:
Done